core/src/de/tobiasroeser/mill/vcs/version/VcsState.scala

```diff
@@ -52,6 +58,28 @@ case class VcsState(
       case t => t
     }
 
+  @deprecated("Binary compatibility shim. Use other overload instead.", "mill-vcs-verison after 0.2.0")
+  private[version] def format(
```

Comment:
Maybe a naive question, but is this needed? With the new change it won't break compat for existing users right because it has a default value?

Reply:
Typically, you're right, as most Mill plugins are used in build scripts, hence their use site is always recompiled, when the version changes. 

But with your new mill-ci-release plugin, we started to have downstream users who depend on binary published artifacts. And there is a high change of really breaking the build, when the user is also explicitly importing both plugins.

Reply:
Ahhh, so you're saying it's my fault! 😆 

Kidding, makes sense.